package.json

```diff
@@ -50,7 +50,7 @@
   },
   "peerDependencies": {
     "@types/react": "^15.0.0 || ^16.0.0",
-    "react": "^15.0.0 || ^16.0.0"
+    "react": "^15.0.0 || ^16.0.0 || ^17.0.0"
```

Comment:
I believe 15 isn‘t working, so might as well update that: https://github.com/remarkjs/react-markdown/issues/313

Can we switch to `>= 16`, so that we’re not bugged by React 18 either? (If 18 breaks, I’m sure we’ll get issues about that and can fix it)